### ISSUE TITLE
Prepare basic hello triangle

### ISSUE THREAD

**Issue by Lut99** [Lut99/Rasterizer#1]
Before we can do anything remotely interesting, we first have to re-structure the project to render something simple like a triangle again. We'll be basing our Vulkan interface on that of the [RayTracer-3](github.com/Lut99/RayTracer-3), and thus aim to have a (somewhat) high-level API which we use for the Vulkan functions.

**Comment by Lut99** [comment]
Let's push model loading to a new issue
